docs/decisions/09-formatting.md

```diff
@@ -0,0 +1,27 @@
+# Formatting
+
+We want to lean into existing tools to take the argument and thought out of applying formatting. It can happen automatically, avoiding nitpicking.
+
+## EditorConfig
```

Comment:
```suggestion
# Formatting

Establishing some minimal style enforcement

## Status

- [ ] Decided
- [ ] Decided Against
- [ ] Deferred
- [ ] Superseded

# Decision

We want to lean into existing tools to take the argument and personal tastes out of applying formatting. This helps to avoid nitpicking, especially if it happens automatically.

## EditorConfig
```

Reply:
This is mostly to keep the ADR doc consistent with the others. It's possible that in the future we'll want to introduce some tooling around them, and consistency will help with that.